app/resources/v1/users.js

```diff
@@ -294,6 +310,7 @@ exports.get = async function (req, res) {
     // seems odd to have these two functions chained together, when we
     // have an ORM that can retrive the user details. is this code still used?
     const result = await findUserById(userId)
+    await BTPTokenCheck(result)
```

Comment:
it looks like doing this here may be breaking user tests 

https://github.com/streetmix/streetmix/runs/2513499007?check_suite_focus=true

Maybe it can be done somewhere else, or maybe the test needs to be adjusted

Reply:
You may need to update the test to mock out the BTPTokenCheck function. I'd recommend moving it to the `../services/integrations/coil` module so that you can more effectively mock this particular file and method from the user test.

Reply:
I could use a clarity on what you mean here. Are you suggesting I could move that function to the other module and just call it here in the same place? Or something else (like chaining it on the api routes instead)? It's a bit of a design question that I wasn't sure how best to address.

I'll go ahead and move the function out, but note there are a couple disadvantages I was concerned about moving this into it's own module and chaining it together on the get: 
* creating redundant database lookups ( maybe not too big of a deal if cached? but we're basically looking up the same user data several times)

* running the BTP check for every user instead of checking if they have a coil connection and then doing the check (honestly not sure if doing the check as part of the user get endpoint vs a check inside the function makes much difference)